src/gsi/read_l2bufr_mod.f90

```diff
@@ -580,106 +580,114 @@ subroutine radar_bufr_read_all(npe,mype)
 		nobs_badsr=nobs_badsr+1
 		cycle
 	     end if
-	     irbin=ceiling(range*rdelr)
-	     if(irbin<1) then
-		nobs_lrbin=nobs_lrbin+1
-		cycle
-	     end if
-	     if(irbin>nrbin) then
-		nobs_hrbin=nobs_hrbin+1
-		cycle
-	     end if
-
-       if (.not.radar_box) then 
-	        iloc=nrbin*(nazbin*(ielbin-1)+(iazbin-1))+irbin
-	        bins(1,iloc,krad)=bins(1,iloc,krad)+range
-	        bins(2,iloc,krad)=bins(2,iloc,krad)+stn_az
-	        bins(3,iloc,krad)=bins(3,iloc,krad)+stn_el
-	        bins(4,iloc,krad)=bins(4,iloc,krad)+rwnd(2,i)
-	        bins(5,iloc,krad)=bins(5,iloc,krad)+rwnd(2,i)**2
-	        bins(6,iloc,krad)=bins(6,iloc,krad)+t
-	        ibins(iloc,krad)=ibins(iloc,krad)+1
+       if (.not.radar_box) then
```

Comment:
Line 583 needs to align with line 582, then indent through line 692.  Please see 

/gpfs/dell2/emc/modeling/noscrub/Michael.Lueken/src/gsi/read_l2bufr_mod.f90

for what the change should look like

Reply:
Committed changes back to the PR.